spec/validators/postcode_validator_spec.rb

```diff
@@ -0,0 +1,106 @@
+require "spec_helper"
+
+class SomeModelWithPostcode
```

Comment:
We can replace this when we have a `Postcode` model, which we'll probably introduce later